doc/classes/RigidBody2D.xml

```diff
@@ -176,6 +176,28 @@
 		<member name="inertia" type="float" setter="set_inertia" getter="get_inertia" default="0.0">
 			The body's moment of inertia. This is like mass, but for rotation: it determines how much torque it takes to rotate the body. The moment of inertia is usually computed automatically from the mass and the shapes, but this property allows you to set a custom value.
 			If set to [code]0[/code], inertia is automatically computed (default value).
+			[b]Note:[/b] This value does not change when inertia is automatically computed. Use [PhysicsServer2D] to get the computed inertia.
+			[codeblocks]
+			[gdscript]
+			@onready var ball = $Ball
+
+			func get_ball_inertia():
+			    return 1 / PhysicsServer2D.body_get_direct_state(ball).inverse_inertia
```

Comment:
Here you're passing the `ball` node reference to `body_get_direct_state`, but in C# you pass the RID. Which is correct?

Reply:
The method signature takes `RID`:

https://github.com/godotengine/godot/blob/bdad9770d64914da3b77bee49916419b5df87d1c/servers/physics_server_2d.h#L484

But I don't know if GDScript does some magic in the bindings so you can pass the object and the C++ receives its `RID`.

Reply:
GDScript automatically passes the RID. I've tested both scripts in the editor and they both work the same.

Reply:
I added the `get_rid()` method to the sample to avoid confusion.